ddr/ddr/Models/GameEntities/Oncomer.swift

```diff
@@ -62,19 +92,15 @@ class Oncomer: SKSpriteNode, Spawnable {
   }
   
   func collidesWith(position: CGPoint) -> Bool {
-    return false
+    return OncomerPosition.of(x: self.position.x) == OncomerPosition.of(x: position.x)
```

Comment:
Again, if you are storing the third of the screen that the oncomer is on then you should use the stored value and not recalculate it here.